config.go

```diff
@@ -19,19 +19,24 @@ type config struct {
 	RescueProxyAPIAddr string
 	RocketscanAPIURL   string
 	AllowedOrigins     []string
+	SecureGRPC         bool
 	Debug              bool
 }
 
 // Check that URL is valid.
-func checkURL(data string) error {
+func checkURL(data string, allowedSchemes []string) error {
```

Comment:
```suggestion
func checkURL(data string, allowedSchemes ...string) error {
```

really minor but using a variadic function here makes invocation cleaner and ensures that the list is established at compile time instead of runtime